recipes/zephyr-7b-beta/README.md

```diff
@@ -9,7 +9,7 @@ As described in the Zephyr [technical report](https://huggingface.co/papers/2310
 See below for commands to train these models using either DeepSpeed ZeRO-3 or LoRA.
 
 ## Full training examples
-
+You will require 8 GPUs (80GB of VRAM) to train the full model.
```

Comment:
Happy to keep this line in the PR if you don't mind reverting the config changes :)

Reply:
haha, my bad cause this is specified on the main readme file =)

Reply:
thanks for iterating!